authors/apps/articles/views.py

```diff
@@ -94,3 +85,72 @@ def update(self, request, slug,*args,**kwargs):
         serializer.is_valid(raise_exception=True)
         serializer.save()
         return Response(serializer.data)
+
+
+User = get_user_model()
```

Comment:
@ManuelDominic , could you elaborate on the use of this line.

Reply:
i was assigning the variable User to the user model, however, since it was already imported, i am going to get rid of it. Thanks